src/http_api/api_types.rs

```diff
@@ -106,3 +106,12 @@ pub struct LedgerBalanceResponse {
 pub struct GnosisSafeConfigResponse {
     pub config: GnosisSafeConfig,
 }
+
+/// Mint Txs
+#[derive(Serialize, Debug, Eq, PartialEq)]
+#[allow(missing_docs)]
+pub struct MintInfoResponse {
+    pub mint_txs: Vec<MintTx>,
+    pub mint_config_txs: Vec<MintConfigTx>,
+    pub mint_configs: Vec<MintConfig>
```

Comment:
I think it would be better to return a response that nets the mint configs under their respective mint config tx. A mint config on its own is not particularly useful since you have no idea which token it is for, for example. This would require creating some other struct above that is something like `MintConfigTxWithConfigs` that contains a `MintConfigTx` and `Vec<MintConfig>`